tests/test_client.py

```diff
@@ -0,0 +1,143 @@
+import unittest
+from unittest.mock import mock_open, patch, call
+from client.client import VariantAnnotationClient, EnsemblError
+from tests import util
+
+TEST_FILENAME = "variants.txt"
+
+TEST_URL = "https://test.com/{variant}"
+
+class TestVariantAnnotationClient(unittest.TestCase):
```

Comment:
thanks for implementing these tests. Feel like they could be mocked more directly to the expected payload, but below works too. Often find https://vcrpy.readthedocs.io/en/latest/ helpful when developing against an API